docs/mesos/README.md

```diff
@@ -45,10 +45,13 @@ The [Calico-Mesos RPM Installation Guide](RpmInstallCalicoMesos.md) serves as th
 ### b.) Manual
 For an in-depth walkthrough of the full installation procedure performed by the RPMs, see the [Calico-Mesos Manual Install Guide](ManualInstallCalicoMesos.md).
 
-### c.) Dockerized Deployment
+### c.) Dockerized Deployment (Automated)
```

Comment:
I think I'll work on it on my train ride home when  i have a clear head. It would affect the intro section of all the guides mentioned, I'll open up a seperate PR for it after